microprofile-fault-tolerance/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source
+  ~ Copyright 2019, Red Hat, Inc. and/or its affiliates, and individual
+  ~ contributors by the @authors tag. See the copyright.txt in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<arquillian xmlns="http://jboss.org/schema/arquillian" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian
+    http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+  <!-- Uncomment to have test archives exported to the file system for inspection -->
+  <!-- <engine> -->
+  <!-- <property name="deploymentExportPath">target/</property> -->
+  <!-- </engine> -->
+
+  <!-- Example configuration for a managed ${product.name} instance -->
+  <container qualifier="jboss" default="true">
+    <!-- By default, Arquillian will use the JBOSS_HOME environment variable to find the ${product.name} installation.
+         If you prefer not to define the JBOSS_HOME environment variable, alternatively you can uncomment the
+         following `jbossHome` property and replace ${jboss.home.name} with the path to your ${product.name} installation. -->
+
+      <!-- In order to run tests, commend out "jbossHome" line -->
+      <configuration>
+          <property name="serverConfig">standalone-microprofile.xml</property>
+          <!--<property name="jbossHome">${jboss.home.name}</property>-->
+      </configuration>
+  </container>
+</arquillian>
```

Comment:
We no longer use ARQ. Remove this as this won't work when testing against deployment running in e.g. OpenShift.